queries/liquid/injections.scm

```diff
@@ -0,0 +1,18 @@
+((template_content) @injection.content
+  (#set! injection.language "html")
+  (#set! injection.combined))
+
+((json_content) @injection.content
+  (#set! injection.language "json")
+  (#set! injection.combined))
+
+((style_content) @injection.content
+  (#set! injection.language "css")
+  (#set! injection.combined))
+
+((js_content) @injection.content
+  (#set! injection.language "javascript")
+  (#set! injection.combined))
```

Comment:
I see that it's inside a `javascript_statement` node, so I think it should be
```suggestion
(javascript_statement
  (js_content) @injection.content
  (#set! injection.language "javascript")
  (#set! injection.combined))
```

Otherwise all js_content from all js blocks are combined into one, which I don't think is the original intention?

Reply:
Good point, I didn't consider this. This applies to the others as well so will adjust.

Reply:
Edit: I just uodated the suggested query @hank. On stable version of neovim injection.combined doesn't really worked, so it's better to comment the query and re-enable it once 0.10 becomes stable

Reply:
lol, ok sounds good.